tools/autograd/gen_inplace_or_view_type.py

```diff
@@ -0,0 +1,268 @@
+from tools.codegen.api import cpp
+from tools.codegen.api.autograd import *
```

Comment:
I can fix all the new ones introduced in this PR and send out a followup PR to clean up the folder later ;) 
But I didn't find a way to add this in flake8 so it'll be hard to prevent future changes :P